website/docs/docs/deploy/ci-jobs.md

```diff
@@ -40,6 +39,11 @@ To make CI job creation easier, many options on the **CI job** page are set to d
     - **dbt compare**<Lifecycle status="enterprise" /> &mdash; Enable this option to compare the last applied state of the production environment (if one exists) with the latest changes from the pull request, and identify what those differences are. To enable record-level comparison and primary key analysis, you must add a [primary key constraint](/reference/resource-properties/constraints) or [uniqueness test](/reference/resource-properties/data-tests#unique). Otherwise, you'll receive a "Primary key missing" error message in dbt Cloud.
     
       To review the comparison report, navigate to the [Compare tab](/docs/deploy/run-visibility#compare-tab) in the job run's details. A summary of the report is also available from the pull request in your Git provider (see the [CI report example](#example-ci-report)). 
+
+      :::info Optimization tip 
+      When you enable the **dbt compare** checkbox, you can customize the comparison command to optimize your CI job. For example, if you have large models that take a long time to compare, you can exclude them to speed up the process using the [`--exclude` flag](/reference/node-selection/exclude). Use [custom commands](/docs/deploy/job-commands#compare-changes-custom-commands) to modify the selection criteria.
+      :::
```

Comment:
ideally, i'd like to link to the event_time page that was created in this [pr here](https://github.com/dbt-labs/docs.getdbt.com/pull/6384) but i can add to it after it's merged

Reply:
also i know it's models/sources/seeds, etc. but once i link to the new event_time page, i'll remove the 'models' mention.